.circleci/config.yml

```diff
@@ -87,8 +87,8 @@ commands:
           name:  "Install Rust"
           command: |
             curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain $(< rust-toolchain) && source $HOME/.cargo/env
-            rustup install nightly-2019-07-31
-            rustup component add rustfmt --toolchain nightly-2019-07-31
+            rustup install nightly-2020-01-15
```

Comment:
Ah yeah, we should fix this by not install something here at all :/

The makefile will actually install Rust correctly if rustup is available. Do you think you can change the CI scripts to just use that so we don't forget that in the future and the makefile is the single source of truth?

Can be a follow-up PR though, this should get in ASAP.

Reply:
It is a bit tricky because of caching, we should discard the cache on update but we can't get the knowledge out of the makefile ..